sacred/commandline_options.py

```diff
@@ -28,6 +29,9 @@ def __init__(
         long_flag: str,
         is_flag: bool,
     ):
+
+        assert re.match(r"-\w", short_flag)
```

Comment:
I wouldn't use `assert` here. I think it is rather uncommon and can be optimized away. Rather raise and explicit `ValueError` with a descriptive message.

Reply:
Done